--- pingus-0.6.0/configure.ac~	Mon Jul 26 01:16:40 2004
+++ pingus-0.6.0/configure.ac	Mon Jul 26 01:16:40 2004
@@ -50,7 +50,7 @@
 
 if test "x$with_gettext" != "xno"; then
 	dnl -- gettext macros --
-	ALL_LINGUAS="cs de fr sr tr" # it nl
+	ALL_LINGUAS="cs de fr sr@Latn tr" # it nl
 
 	dnl AM_WITH_NLS
 	dnl AM_LC_MESSAGES
--- pingus-0.6.0/configure~	Mon Jul 26 01:16:26 2004
+++ pingus-0.6.0/configure	Mon Jul 26 01:16:26 2004
@@ -3553,7 +3553,7 @@
 fi;
 
 if test "x$with_gettext" != "xno"; then
-		ALL_LINGUAS="cs de fr sr tr" # it nl
+		ALL_LINGUAS="cs de fr sr@Latn tr" # it nl
 
 
       if test "X$prefix" = "XNONE"; then
