diff -urN pingus-0.6.0.orig/src/blitter.hxx pingus-0.6.0/src/blitter.hxx
--- pingus-0.6.0.orig/src/blitter.hxx	Tue Jun  3 23:42:03 2003
+++ pingus-0.6.0/src/blitter.hxx	Tue Jun  3 23:50:49 2003
@@ -20,6 +20,7 @@
 #ifndef HEADER_PINGUS_BLITTER_HXX
 #define HEADER_PINGUS_BLITTER_HXX
 
+#include <cassert>
 #include "pingus.hxx"
 
 class CL_Canvas;
diff -urN pingus-0.6.0.orig/src/col_map.hxx pingus-0.6.0/src/col_map.hxx
--- pingus-0.6.0.orig/src/col_map.hxx	Tue Jun  3 23:42:03 2003
+++ pingus-0.6.0/src/col_map.hxx	Tue Jun  3 23:52:19 2003
@@ -20,6 +20,7 @@
 #ifndef HEADER_PINGUS_COL_MAP_HXX
 #define HEADER_PINGUS_COL_MAP_HXX
 
+#include <cassert>
 #include "groundtype.hxx"
 
 class GraphicContext;
diff -urN pingus-0.6.0.orig/src/console.hxx pingus-0.6.0/src/console.hxx
--- pingus-0.6.0.orig/src/console.hxx	Tue Jun  3 23:42:03 2003
+++ pingus-0.6.0/src/console.hxx	Tue Jun  3 23:54:52 2003
@@ -20,6 +20,7 @@
 #ifndef HEADER_PINGUS_CONSOLE_HXX
 #define HEADER_PINGUS_CONSOLE_HXX
 
+#include <cassert>
 #include <string>
 #include <iostream>
 #include "gui/display.hxx"
diff -urN pingus-0.6.0.orig/src/demo_recorder.hxx pingus-0.6.0/src/demo_recorder.hxx
--- pingus-0.6.0.orig/src/demo_recorder.hxx	Tue Jun  3 23:42:03 2003
+++ pingus-0.6.0/src/demo_recorder.hxx	Tue Jun  3 23:56:32 2003
@@ -20,6 +20,7 @@
 #ifndef HEADER_PINGUS_DEMO_RECORDER_HXX
 #define HEADER_PINGUS_DEMO_RECORDER_HXX
 
+#include <cassert>
 #include "pingus.hxx"
 #include <fstream>
 #include <string>
diff -urN pingus-0.6.0.orig/src/editor/generic_property_frame.hxx pingus-0.6.0/src/editor/generic_property_frame.hxx
--- pingus-0.6.0.orig/src/editor/generic_property_frame.hxx	Tue Jun  3 23:42:03 2003
+++ pingus-0.6.0/src/editor/generic_property_frame.hxx	Tue Jun  3 23:43:00 2003
@@ -20,6 +20,7 @@
 #ifndef HEADER_PINGUS_GENERIC_PROPERTY_FRAME_HXX
 #define HEADER_PINGUS_GENERIC_PROPERTY_FRAME_HXX
 
+#include <cassert>
 #include <vector>
 #include <string>
 #include <ClanLib/GUI/button.h>
diff -urN pingus-0.6.0.orig/src/gui/root_gui_manager.hxx pingus-0.6.0/src/gui/root_gui_manager.hxx
--- pingus-0.6.0.orig/src/gui/root_gui_manager.hxx	Tue Jun  3 23:42:03 2003
+++ pingus-0.6.0/src/gui/root_gui_manager.hxx	Tue Jun  3 23:43:44 2003
@@ -20,6 +20,7 @@
 #ifndef HEADER_PINGUS_GUI_ROOT_GUI_MANAGER_HXX
 #define HEADER_PINGUS_GUI_ROOT_GUI_MANAGER_HXX
 
+#include <cassert>
 #include "gui_manager.hxx"
 
 namespace Input
diff -urN pingus-0.6.0.orig/src/pingu_holder.hxx pingus-0.6.0/src/pingu_holder.hxx
--- pingus-0.6.0.orig/src/pingu_holder.hxx	Tue Jun  3 23:42:03 2003
+++ pingus-0.6.0/src/pingu_holder.hxx	Tue Jun  3 23:58:34 2003
@@ -20,6 +20,7 @@
 #ifndef HEADER_PINGUS_PINGU_HOLDER_HXX
 #define HEADER_PINGUS_PINGU_HOLDER_HXX
 
+#include <cassert>
 #include <list>
 #include <vector>
 #include "worldobj.hxx"
diff -urN pingus-0.6.0.orig/src/savegame_manager.hxx pingus-0.6.0/src/savegame_manager.hxx
--- pingus-0.6.0.orig/src/savegame_manager.hxx	Tue Jun  3 23:42:03 2003
+++ pingus-0.6.0/src/savegame_manager.hxx	Wed Jun  4 00:00:40 2003
@@ -20,6 +20,7 @@
 #ifndef HEADER_SAVEGAME_MANAGER_HXX
 #define HEADER_SAVEGAME_MANAGER_HXX
 
+#include <cassert>
 #include <map>
 #include "savegame.hxx"
 
diff -urN pingus-0.6.0.orig/src/worldmap/dot.hxx pingus-0.6.0/src/worldmap/dot.hxx
--- pingus-0.6.0.orig/src/worldmap/dot.hxx	Tue Jun  3 23:42:03 2003
+++ pingus-0.6.0/src/worldmap/dot.hxx	Tue Jun  3 23:47:53 2003
@@ -20,6 +20,7 @@
 #ifndef HEADER_PINGUS_DOT_HXX
 #define HEADER_PINGUS_DOT_HXX
 
+#include <cassert>
 #include <string>
 #include "../libxmlfwd.hxx"
 #include "../vector.hxx"
