--- pingus-0.7.0/src/SConscript.orig	2007-08-24 23:49:45.000000000 +0200
+++ pingus-0.7.0/src/SConscript	2007-09-14 21:59:52.759075168 +0200
@@ -21,8 +21,9 @@
 import os
 
 env = Environment(# ENV = {'PKG_CONFIG_PATH' : os.getenv('PKG_CONFIG_PATH')},
-                  CCFLAGS = ['-O2', '-Wall', '-Werror', '-g'],
-                  CPPDEFINES = ['ENABLE_BINRELOC'],
+		  CXX = ARGUMENTS.get('CXX', 'g++'),
+		  CC = ARGUMENTS.get('CC', 'gcc'),
+                  CCFLAGS = ARGUMENTS.get('CXXFLAGS', ['-O2', '-Wall', '-Werror', '-g']),
                   CPPPATH = ['..', '.'])
 
 env.ParseConfig('sdl-config  --cflags --libs')
--- pingus-0.7.0/src/pingus_main.cpp.orig	2007-09-14 21:53:32.829424211 +0200
+++ pingus-0.7.0/src/pingus_main.cpp	2007-09-14 21:53:19.504664876 +0200
@@ -627,6 +627,7 @@ PingusMain::init_path_finder()
   path_manager.add_path(data_path);
   path_manager.add_path("data");
 #else
+  path_manager.add_path("/usr/share/pingus/data");
   path_manager.add_path("data");
 #endif
 
--- pingus-0.7.0/SConstruct.orig	2005-12-05 18:00:26.000000000 +0100
+++ pingus-0.7.0/SConstruct	2007-09-14 21:43:22.494643214 +0200
@@ -22,6 +22,7 @@ import sys, os
 
 opts = Options(['options.cache', 'custom.py'])
 opts.Add('CXX', 'The C++ compiler')
+opts.Add('CXXFLAGS', 'compiler flags')
 opts.Add('PKG_CONFIG_PATH', 'Path for pkg-config files')
 
 env = Environment(options=opts)
