--- pingus-0.7.1/src/input/SConstruct.main~	2007-09-03 19:17:58.000000000 +0200
+++ pingus-0.7.1/src/input/SConstruct.main	2007-09-23 22:01:17.000000000 +0200
@@ -1,6 +1,8 @@
 ##  -*- python -*-
 
-env = Environment(CCFLAGS  = ['-O0', '-Wall', '-Werror', '-g'],
+env = Environment(CXX = ARGUMENTS.get('CXX', 'g++'),
+		  CC = ARGUMENTS.get('CC', 'gcc'),
+		  CCFLAGS  = ARGUMENTS.get('CXXFLAGS', ['-O2', '-Wall', '-Werror', '-g']),
                   CPPPATH  = ['..', '../..'],
                   CPPDEFINES = ['HAVE_CWIID'],
                   LIBS = ['cwiid'])
--- pingus-0.7.1/src/pingus_main.cpp~	2007-09-22 17:58:45.000000000 +0200
+++ pingus-0.7.1/src/pingus_main.cpp	2007-09-23 22:04:32.000000000 +0200
@@ -583,6 +583,7 @@
       path_manager.add_path("data"); // assume game is run from source dir, without any magic
       free(exe_path);
 #else
+      path_manager.add_path("/usr/share/pingus/data");
       path_manager.add_path("data"); // assume game is run from source dir
 #endif
 
--- pingus-0.7.1/SConstruct~	2007-09-22 17:46:31.000000000 +0200
+++ pingus-0.7.1/SConstruct	2007-09-23 22:07:59.000000000 +0200
@@ -302,6 +302,7 @@
    opts.Add('LINKFLAGS', 'Linker Compiler flags', [])
    opts.Add('CC', 'C Compiler', 'gcc')
    opts.Add('CXX', 'C++ Compiler', 'g++')
+   opts.Add('CXXFLAGS', 'compiler flags')
 #   opts.Add('debug', 'Build with debugging options', 0)
 #   opts.Add('profile', 'Build with profiling support', 0)
    opts.Add('with_xinput', 'Build with Xinput support', False)
