--- pingus-0.5.0pre3/src/backgrounds/StarfieldBackground.cc.orig	Tue Jan 15 11:48:50 2002
+++ pingus-0.5.0pre3/src/backgrounds/StarfieldBackground.cc	Sat Oct 12 16:54:23 2002
@@ -185,7 +185,7 @@
 }
 
 void
-StarfieldBackgroundStars::draw_offset(int x_of, int y_of, float s = 1.0)
+StarfieldBackgroundStars::draw_offset(int x_of, int y_of, float s)
 {
   sur.put_screen (int(x_pos + x_of), int(y_pos + y_of));
 }
@@ -231,7 +231,7 @@
 }
 ///
 void 
-StarfieldBackground::draw_offset(int x_of, int y_of, float s = 1.0)
+StarfieldBackground::draw_offset(int x_of, int y_of, float s)
 {
   for (std::vector<StarfieldBackgroundStars>::iterator i = stars.begin ();
        i != stars.end (); i++)
